### ISSUE TITLE
Recipe view needs to conform to yummly's requirements

### ISSUE THREAD

**Issue by white0ut** [White0ut/PantryTracker#1]
As you can see, the recipe fragment isn't that attractive, due to my lack of layout chops. Before the app can be pushed anywhere, I will have to conform to the yummly api [Attribution Requirements](https://developer.yummly.com/documentation#Attribution).  I'd love to see the app at least conform to the bare minimum so I can push something out.  Then I can start to get up to par with the design chops Yummly has!

**Comment by JoshKlingler** [comment]
Yeah sure I can take a look at it! It's been a while since I've done any
Android stuff but I'll see what I can do.

**Comment by white0ut** [comment]
Haha, not necessarily for you specifically, I have some buddies who said
they would look at it also :) But thank you!

On Thu, May 7, 2015 at 8:02 AM, Josh Klingler notifications@github.com
wrote:

> Yeah sure I can take a look at it! It's been a while since I've done any
> Android stuff but I'll see what I can do.
> 
> On Wed, May 6, 2015 at 11:31 PM, Kendrick Cline notifications@github.com
> wrote:
> 
> > As you can see, the recipe fragment isn't that attractive, due to my lack
> > of layout chops. Before the app can be pushed anywhere, I will have to
> > conform to the yummly api Attribution Requirements
> > https://developer.yummly.com/documentation#Attribution. I'd love to
> > see
> > the app at least conform to the bare minimum so I can push something out.
> > Then I can start to get up to par with the design chops Yummly has!
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/White0ut/PantryTracker/issues/1.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/White0ut/PantryTracker/issues/1#issuecomment-99833977
> .

## 

_Kenny Cline_ | Software Engineer

(207) 798-0903​

@Kendrick_Cline

​

maestro· http://www.meetmaestro.com/

_O_ 401 E. MICHIGAN AVENUE SUITE 202 KALAMAZOO, MI 49007
_P_ (800) 319-2122   _F_ (207) <(269)%20585-4019>798-0903   _W_
MEETMAESTRO.COM http://www.meetmaestro.com/

CONFIDENTIALITY NOTICE: This e-mail message, including any attachments, is
for the sole use of the intended recipients(s) and may contain confidential
and privileged information. Any unauthorized review, use, disclosure or
distribution is prohibited. If you are not the intended recipient, please
contact the sender by reply e-mail and destroy all copies of the original
message.​

**Comment by white0ut** [comment]
Actually, I'll call him out... @dillonburton
